src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import { Partner } from '../Partner/Partner';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age } = person;
+
+  return (
+    <>
```

Comment:
You don`t need jsx fragment in this case ( you have section as parent tag element)

Reply:
done